api/src/test/resources/validCdaLevel3Sample.xml

```diff
@@ -86,7 +86,10 @@
 			<addr/>
 			<telecom/>
 			<associatedPerson>
-				<name>NextOfKin(NOK) Name</name>
+                <name>
```

Comment:
Openmrs requires both a given and family name, was the intent for this to let the cda have 2 name components (given and family both) specified inside a single name tag? Or to allow just one name i.e given or family